### ISSUE TITLE
All Zip Files are Corrupt

### ISSUE THREAD

**Issue by GoogleCodeExporter** [marutichintan/pymssql#1]
```
What steps will reproduce the problem?
1. Download any zip file from "Downloads".
2. Attempt to unzip.
3. Bash head on keyboard.

What is the expected output? What do you see instead?
I expect the file to unzip.  However, the zip files are all obviously 
incomplete as they are less than 1kb.

What version of the product are you using? On what operating system?
1.9.906

Please provide any additional information below.
Don't upload broken zipfiles.

```

Original issue reported on code.google.com by `bluedrag...@gmail.com` on 13 Apr 2010 at 9:48

**Comment by GoogleCodeExporter** [comment]
```
I downloaded pretty fine some minutes ago. A problem are occuring during 
install:
gcc: _mssql.c: Arquivo ou diretório não encontrado
gcc: no input files
error: command 'gcc' failed with exit status 1

I have all the preReqs.
Júlio
```

Original comment by `julio.010101@gmail.com` on 13 Apr 2010 at 9:55

**Comment by GoogleCodeExporter** [comment]
```
Well, perhaps I should be more specific...any zip files but the source-only zip 
file.  
That includes the egg and exe files.
```

Original comment by `bluedrag...@gmail.com` on 13 Apr 2010 at 9:59

**Comment by GoogleCodeExporter** [comment]
```
pymssql-1.9.906.win32.zip in Downloads is corrupt.
```

Original comment by `akimov.alex` on 14 Apr 2010 at 6:17

**Comment by GoogleCodeExporter** [comment]
```
While I appreciate the fact this package exists at all, I think it's quite
irresponsible to upload something like this, where it will break the 
installation of
any packages that depend on it (as mine do). Probably there is a problem with 
PyPi as
well, that it allows packages to be updated without verifying that the content 
is at
least marginally useful.
```

Original comment by `alecmu...@gmail.com` on 14 Apr 2010 at 2:52

**Comment by GoogleCodeExporter** [comment]
```
I have not been able to get it to install out of PyPi, either.  Same issues.
```

Original comment by `bluedrag...@gmail.com` on 14 Apr 2010 at 4:38

**Comment by GoogleCodeExporter** [comment]
```
Apologies, I didn't think to check the binary packages. Windows isn't my 
primary 
platform so I was mainly concentrating on the source builds.

If someone else wishes to step up and handle building and packaging on Windows 
that 
would be more than welcome!
```

Original comment by `dam...@gmail.com` on 14 Apr 2010 at 10:04

**Comment by GoogleCodeExporter** [comment]
```
I'll take care of this in the future. Sorry for any inconvenience caused by this
issue, the .exe file is now ok and is available for download. .egg in progress.
BEWARE that you may experience other problems -- see the release notes (this is
pre-release of pymssql 2.0 which will contain some groundbreaking changes, you
definitely want to know about them).
```

Original comment by `akuk...@gmail.com` on 14 Apr 2010 at 10:56
- Changed state: **Done**
- Added labels: **Component-Scripts**, **OpSys-Windows**

**Comment by GoogleCodeExporter** [comment]
```
Thanks a lot, and that's perfectly fine - I'm willing to be a guinea pig on 
bleeding 
edge software releases.
```

Original comment by `bluedrag...@gmail.com` on 15 Apr 2010 at 2:37

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the fix. I think we will pin to an earlier version to avoid any
instability for now (we barely use it anyway, so we don't need any of the new 
goodies).
```

Original comment by `alecmu...@gmail.com` on 16 Apr 2010 at 10:33

**Comment by GoogleCodeExporter** [comment]
```
I've tried both
http://pymssql.googlecode.com/files/pymssql-1.9.906.tar.bz2
http://pymssql.googlecode.com/files/pymssql-1.9.906.tar.gz
today (2010-04-21). Build fails with "gcc: _mssql.c: No such file or directory".
```

Original comment by `maxim.go...@gmail.com` on 21 Apr 2010 at 12:44

**Comment by GoogleCodeExporter** [comment]
```
I can't find _mssql.c as well!
I tried either on Mac either on Ubuntu
```

Original comment by `cambria....@gmail.com` on 21 Apr 2010 at 2:16

**Comment by GoogleCodeExporter** [comment]
```
The issue is hopefully fixed in pymssql version 1.9.907. Please download and 
try again.
```

Original comment by `akuk...@gmail.com` on 21 Apr 2010 at 3:26

**Comment by GoogleCodeExporter** [comment]
```
hi, I have recently downloaded pymssql 1.9.907. It is still giving the same 
problem
gcc: _mssql.c: No such file or directory
```

Original comment by `gaurav.k...@gmail.com` on 27 Apr 2010 at 2:48

**Comment by GoogleCodeExporter** [comment]
```
Please try the freshly released pymssql 1.9.908. The issue should be gone for 
good,
finally.
```

Original comment by `akuk...@gmail.com` on 28 Apr 2010 at 11:51
